zipline/assets/asset_writer.py

```diff
@@ -165,6 +165,62 @@ def _generate_output_dataframe(data_subset, defaults):
     return output
 
 
+# Define a version number for the database generated by these writers
+# Increment this version number any time a breaking change is made to the
+# schema and readers of the database
+ASSET_DB_VERSION = 1
+
+
+def check_version_info(version_table, expected_version, write_version=False):
+    """
+    Checks for a version value in the version table, and inserts version
+    info if none is present.
+
+    Parameters
+    ----------
+    version_table : sa.Table
+        The version table of the asset database
+    expected_version : int
+        The expected version of the asset database
+    write_version : bool, optional
```

Comment:
When initializing the database -- the AssetDBWriter will write the version in to the table if it is not present.